pkg/webhook/resources/setting/validator.go

```diff
@@ -401,6 +408,70 @@ func (v *settingValidator) customizeTransport() error {
 	return nil
 }
 
+func validateNTPServers(setting *v1beta1.Setting) error {
```

Comment:
seems different than harvester-installer,  and the NTP server is not trully validated if it works or not.

what's the consideration?

https://github.com/harvester/harvester-installer/blob/master/pkg/console/util.go#L101

Reply:
Yap, I checked the harvester installer before I implemented the validator. I skip this check because we already have the monitoring mechanism on node-manager. Also, if the NTP server leaves temporarily, that should not pop an error to the user. So for this scenario, I prefer to do the monitoring by `node-manager`.

Reply:
If the NTP server IP is wrong, the validator should reject, rather than let `node-manager` report error later.

When validator does not report error, user may assume the configure is successful.

If the server is temporary offline, the `reject` in configuration is still acceptable. User need to wait until the server is up and then reconfigure it.